package.json

```diff
@@ -8,10 +8,15 @@
     "node": ">=6.13.0"
   },
   "scripts": {
+    "start": "nodemon server/index.js"
```

Comment:
Now that we're running each other's services on our own proxies, it may make sense to separate our nodemon server script as a dev script and have a separate npm start that turns our server on with node instead.

Reply:
Changed.